packages/tools/generator-widget/generators/app/templates/packages/package_native.json.ejs

```diff
@@ -26,9 +26,6 @@
   "jest-junit": {
     "output": "./dist/testresults/TESTS-Jest.xml"
   },<% } %>
-  "bugs": {
-    "url": "https://github.com/<%- author %>/<%- packageName %>/issues"
-  },
```

Comment:
@tamelbrun, I'm removing this part, because I'm not sure that we can produce good URI. I think it's better to align with web template. What's your opinion on that?

Reply:
I tend to agree, I don't think that the `author` field is suitable to generate a GitHub URL, as there is no guarantee that it even remotely resembles a user name :P.